index.js

```diff
@@ -70,9 +79,24 @@ module.exports = {
         this.log('preparing to activate `' + revisionKey + '`', { verbose: true });
 
         var s3 = new S3({ plugin: this });
-        return s3.activate(options);
+        return Promise.resolve(s3.activate(options))
+          .then(function(){
+            return {
+              revisionData: {
+                activatedRevisionKey: revisionKey
+              }
+            };
+          })
+          .catch(this._errorMessage.bind(this));
```

Comment:
There should already be an error reported to the console without catching here. I think `ember-cli-deploy` already takes care of this. I may be mistaken though.